modules/administration-guide/partials/proc_restoring-a-che-instance-by-chectl-using-a-checluserbackup-custom-object.adoc

```diff
@@ -0,0 +1,35 @@
+[id="restoring-a-{prod-id-short}-instance-by-{prod-cli}-using-a-checluserbackup-custom-object_{context}"]
+= Restoring a {prod-short} instance by {prod-cli} using a CheClusterBackup custom object
+
+//To restore a state described by an existing backup Custom Resource:
+
+If a `CheClusterBackup` custom object is used to create a backup snapshot of a {prod-short} instance, you can restore the {prod-short} instance by entering the name of the `CheClusterBackup` custom object as an argument with the `{prod-cli} server:restore` command:
+
+.Prerequisites
+* Installed `{prod-cli}`.
+* The `CheClusterBackup` custom object that was used to create the backup snapshot.
+* The backup snapshot that was taken using the `CheClusterBackup` custom object and stored on any of the following:
+** xref:backups-of-che-instances-to-an-sftp-backup-server.adoc#backing-up-a-che-instance-using-the-checlusterbackup-custom-object_{context}[External SFTP backup server]
+** xref:backups-of-che-instances-to-amazon-s3.adoc#backing-up-a-che-instance-using-the-checlusterbackup-custom-object_{context}[Amazon S3]
+** xref:backups-of-che-instances-to-a-rest-backup-server.adoc#backing-up-a-che-instance-using-the-checlusterbackup-custom-object_{context}[External REST backup server]
+** xref:backups-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
```

Comment:
Well, if the backup was successful and no one deleted something (the backup server config CR or backup snapshot from the backup server or stop the backup server itself) existence of the snapshot is implied.